src/core/test_helpers/kbn_server.ts

```diff
@@ -107,6 +107,25 @@ export function createRootWithCorePlugins(settings = {}, cliArgs: Partial<CliArg
       username: kibanaServerTestUser.username,
       password: kibanaServerTestUser.password,
     },
+    // Log ES deprecations to surface these in CI
+    logging: {
+      loggers: [
+        {
+          name: 'root',
+          level: 'error',
```

Comment:
before jest integration tests wouldn't log anything to the console, but having at least `error` logs is useful for failing/flaky tests